spring-cloud-alibaba-sentinel-datasource/src/main/java/com/alibaba/cloud/sentinel/datasource/config/NacosDataSourceProperties.java

```diff
@@ -30,7 +30,7 @@
  */
 public class NacosDataSourceProperties extends AbstractDataSourceProperties {
 
-	private String serverAddr;
+	private String serverAddr = "localhost:8848";
```

Comment:
I think it can't be placed here. when invoke the `preCheck()` method,  it always not empty, And will break the previous logic.  Maybe this can be achieved.`getProperty(
					"spring.cloud.sentinel.datasource.nacos.server-addr", "");						"spring.cloud.sentinel.datasource.nacos.server-addr", "localhost:8848")`

Reply:
`NacosDataSourceProperties#serverAddr ` add default value, `preCheck()` Isn't it necessary?